src/main/java/com/epam/spring/hotel/controller/OrderController.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.spring.hotel.controller;
+
+import com.epam.spring.hotel.controller.dto.OrderDto;
+import com.epam.spring.hotel.service.OrderService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping(value = "/order")
```

Comment:
Use @RequestMapping("/order") instead, because you are duplicating same url in each method.